content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -0,0 +1,349 @@
+`kubeadm join phase`.
+
+Para visualizar a lista ordenada de fases e subfases disponíveis, você pode
+executar o comando `kubeadm join --help`. A lista estará localizada no topo da
+tela da ajuda e cada fase terá uma descrição ao lado. Note que ao chamar
+`kubeadm join` todas as fases e subfases serão executadas nesta ordem exata.
+
+Algumas fases possuem opções únicas, portanto, se você desejar ver uma lista das
+opções disponíveis, adicione a _flag_ `--help`. Por exemplo:
+
+```shell
+kubeadm join phase kubelet-start --help
+```
+
+De forma semelhante ao comando
+[`kubeadm init phase`](/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-init/#init-phases),
+`kubeadm join phase` permite que você ignore uma lista de fases utilizando a
+opção `--skip-phases`.
+
+Por exemplo:
+
+```shell
+sudo kubeadm join --skip-phases=preflight --config=config.yaml
+```
+
+{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+
+Alternativamente, você pode utilizar o campo `skipPhases` no manifesto
+`JoinConfiguration`.
+
+### Descobrindo em qual autoridade de certificação (CA) do cluster confiar
+
+A descoberta do kubeadm tem diversas opções, cada uma com suas próprias
+contrapartidas de segurança. O método correto para o seu ambiente depende de
+como você aprovisiona seus nós e as expectativas de segurança que você tem a
+respeito da rede e ciclo de vida dos seus nós.
+
+#### Descoberta baseada em token com fixação da autoridade de certificação (CA)
+
+Este é o modo padrão do kubeadm. Neste modo, o kubeadm baixa a configuração do
+cluster (incluindo a CA raiz) e a valida, utilizando o token, além de verificar
+que a chave pública da CA raiz corresponda ao _hash_ fornecido e que o
+certificado do servidor da API seja válido sob a CA raiz.
+
+O _hash_ da chave pública da CA tem o formato `sha256:<hash_codificado_em_hexa>`.
+Por padrão, o valor do _hash_ é retornado no comando `kubeadm join` impresso ao
+final da execução de `kubeadm init` ou na saída do comando
+`kubeadm token create --print-join-command`. Este _hash_ é gerado em um formato
+padronizado (veja a [RFC7469](https://tools.ietf.org/html/rfc7469#section-2.4))
+e pode também ser calculado com ferramentas de terceiros ou sistemas de
+provisionamento. Por exemplo, caso deseje utilizar a ferramenta de linha de
+comando do OpenSSL:
+
+```shell
+openssl x509 -pubkey -in /etc/kubernetes/pki/ca.crt | openssl rsa -pubin -outform der 2>/dev/null | openssl dgst -sha256 -hex | sed 's/^.* //'
+```
+
+**Exemplos de comandos `kubeadm join`:**
+
+Para nós de processamento:
+
+```shell
+kubeadm join --discovery-token abcdef.1234567890abcdef --discovery-token-ca-cert-hash sha256:1234..cdef 1.2.3.4:6443
+```
+
+Para nós da camada de gerenciamento:
+
+```shell
+kubeadm join --discovery-token abcdef.1234567890abcdef --discovery-token-ca-cert-hash sha256:1234..cdef --control-plane 1.2.3.4:6443
+```
+
+Você pode também rodar o comando `join` para um nó da camada de gerenciamento com
```

Comment:
I prefer the "executar o comando" like line 53.

Reply:
Updated!